extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -491,4 +496,19 @@ private boolean isEqual(State state1, State state2) {
         return colorModeIsEqual && effectIsEqual;
     }
 
+    @Override
+    public Collection<ConfigStatusMessage> getConfigStatus() {
+        // The bridge IP address to be used for checks
+        final String bridgeIpAddress = (String) getThing().getConfiguration().get(HOST);
+
+        // Check whether an IP address is provided
+        if (bridgeIpAddress == null) {
+
+            return ImmutableList.of(ConfigStatusMessage.Builder.error(HOST)
+                    .withMessageKeySuffix(HueConfigStatusMessage.IP_ADDRESS_MISSING.getMessageKey()).withArguments(HOST)
+                    .build());
+        }
```

Comment:
I would not use a regex expression to check for a valid IPv4 address... The IP address is something that can simply be checked by the ```ConfigDescriptionValidator```. For this reason we have to modify the contexts of the config description parameters. Currently there is only the context <i>network-address</i> which is not specific enough to validate it... @kaikreuzer what is your opinion regarding this?

Adding the additional isEmpty check is ok for me...

btw, I would not add the additional else statement, but this is a matter of taste